examples/example_eval_swift_openai_api.py

```diff
@@ -47,16 +47,16 @@ def run_swift_eval():
     task_cfg = dict(
         eval_backend='OpenCompass',
         eval_config={
-            'datasets': ['winogrande'],
+            'datasets': ['gsm8k', 'ARC_c'],
             'models': [
                 {
-                    'path': 'qwen2-7b-instruct',  # Please make sure the model is deployed
+                    'path': 'qwen2.5',  # Please make sure the model is deployed
```

Comment:
The `path` should be aligned with MS/HF model_name, e.g. for Qwen/Qwen2.5-7B-Instruct, the path should be `Qwen2.5-7B-Instruct`

Reply:
done